README.md

```diff
@@ -414,6 +414,7 @@ Table of Contents
   * [instrumentalapp.com](https://instrumentalapp.com) - Beautiful and easy-to-use application and server monitoring with up to 500 metrics and 3 hours of data visibility for free
   * [letsmonitor.org](https://letsmonitor.org) - SSL monitoring, free for up to 5 monitors
   * [loader.io](https://loader.io/) — Free load testing tools with limitations
+  * [netdata.cloud](https://netdata.cloud/) — Free and open-source software, Netdata is distributed, real-time, performance and health monitoring for systems and applications. It is a highly-optimized monitoring agent you install on all your systems and containers.
```

Comment:
This is great, but whats the SaaS angle here?

Reply:
@ripienaar Oh sorry, didn't notice that the list is only and only Iaas, SaaS, and PaaS.